runtime/gc_modron_startup/mmparse.cpp

```diff
@@ -1472,6 +1472,20 @@ gcParseCommandLineAndInitializeWithValues(J9JavaVM *vm, IDATA *memoryParameters)
 	/* Parse the command line 
 	 * Order is important for parameters that match as substrings (-Xmrx/-Xmr)
 	 */
+	{
+		bool eanbleOriginalJDK8HeapSizeCompatibilityOption = false;
+		/* only parse VMOPT_XXENABLEORIGINALJDK8HEAPSIZECOMPATIBILITY option for Java 8 and below */
+		if (J2SE_18 >= J2SE_VERSION(vm)) {
+
+			IDATA eanbled = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXENABLEORIGINALJDK8HEAPSIZECOMPATIBILITY, NULL);
+			IDATA disabled = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXDISABLEORIGINALJDK8HEAPSIZECOMPATIBILITY, NULL);
+			if (eanbled > disabled) {
```

Comment:
wrong spelling of 'enable' at a few spots in this method

Reply:
done